### ISSUE TITLE
Allow to specify DAYS in a different language when parsing structured data

### ISSUE THREAD

**Issue by arrival-spring** [alltheplaces/alltheplaces#10897]
With a StructuredDataSpider, you can specify time_format, but not any different language for days. It would be good to allow that, to save having to re-parse the hours.

For example, see https://www.virginactive.it/club/milano/milano-vittorio-emanuele

```
"openingHoursSpecification": [
    {
    "@type": "OpeningHoursSpecification",
    "dayOfWeek": [
        "Luned&#236; - Venerd&#236;"
        ],
    "opens": "6.30",
    "closes": "23.00"
    }
,
    {
    "@type": "OpeningHoursSpecification",
    "dayOfWeek": [
        "Sabato - Domenica"
        ],
    "opens": "8.00",
    "closes": "20.00"
    }
```

**Comment by Cj-Malone** [comment]
I don't think that's in spec? Anyway the encoding issue would probably mean we have to do something in spider anyway. I vote to try and clean it in pre_process

**Comment by davidhicks** [comment]
> I don't think that's in spec? Anyway the encoding issue would probably mean we have to do something in spider anyway. I vote to try and clean it in pre_process

Yep schema.org's `OpeningHoursSpecification` has a preset enumeration of English day names only per https://schema.org/OpeningHoursSpecification and https://schema.org/DayOfWeek

For every day the feature is typically open there should be something like:
```
    {
      "@type": "OpeningHoursSpecification",
      "closes": "17:00:00",
      "dayOfWeek": "https://schema.org/Monday",
      "opens": "09:00:00"
    },
```

Ranges and lists of days are not supported. Each day has to be individually listed.

Times also have to be listed only in format `hh:mm:ss[Z|(+|-)hh:mm]` (e.g. seconds included).

Perhaps we could add a helper function that takes a non-conformant `openingHoursSpecification` and creates a string of `{dayOfWeek}: {opens} - {closes}` and passes this to `OpeningHours.add_ranges_from_string` with relevant `DAYS_ZZ` for the language applicable, as `OpeningHours.add_ranges_from_string` is purpose designed to parse random opening hours strings to extract whatever it can.
